crates/pathfinder/src/rpc.rs

```diff
@@ -234,6 +240,52 @@ pub fn run_server(addr: SocketAddr, api: RpcApi) -> Result<(HttpServerHandle, So
         let request = params.parse::<NamedArgs>()?.filter;
         context.get_events(request).await
     })?;
+    module.register_async_method(
+        "starknet_addInvokeTransaction",
+        |params, context| async move {
+            #[serde_with::serde_as]
+            #[derive(Debug, Deserialize)]
+            pub struct NamedArgs {
+                pub function_invocation: Call,
+                #[serde_as(as = "Vec<CallSignatureElemAsDecimalStr>")]
+                pub signature: Vec<CallSignatureElem>,
+                #[serde_as(as = "FeeAsHexStr")]
+                pub max_fee: Fee,
+                #[serde_as(as = "TransactionVersionAsHexStr")]
+                pub version: TransactionVersion,
+            }
+            let params = params.parse::<NamedArgs>()?;
+            context
+                .add_invoke_transaction(
+                    params.function_invocation,
+                    params.signature,
+                    params.max_fee,
+                    params.version,
+                )
+                .await
+        },
+    )?;
+    module.register_async_method(
+        "starknet_addDeployTransaction",
+        |params, context| async move {
+            #[serde_with::serde_as]
```

Comment:
Is this needed here, or just forgotten?

Reply:
Ah, just a copy-paste issue I guess. ;)

Removed, thanks!